src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -1,6 +1,13 @@
+import * as _ from 'lodash'
+
 import { getSideArrow } from '../../utils'
-import { pxToRem } from '../../../../lib'
-import { ComponentSlotStyleFunction, ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+import { pxToRem, getColorFromScheme } from '../../../../lib'
```

Comment:
I remember I used `getColorFromScheme` in the renderComponent, so that the user will receive colors object in the styles containing the merged colors. Please take a look on that. If that is not sufficient, we may rework the mechanism there and try to avoid this method in the styles.

Reply:
not relevant anymore, see #756